application/models.py

```diff
@@ -10,7 +10,7 @@
 
 
 class HighSchoolApplication(models.Model):
-    application_number = models.CharField(max_length=10)
+    application_number = models.CharField(max_length=150)
```

Comment:
For this sprint we are just changing the length of the application number rather than changing what the application number consists of?

Reply:
yes, for now, the application number consists of a combination of user id, school and program. The school object is getting displayed instead of school name bug will be fixed by Abe.